app/src/main/java/ca/bc/gov/bchealth/ui/home/manage/QuickAccessManagementScreen.kt

```diff
@@ -0,0 +1,189 @@
+package ca.bc.gov.bchealth.ui.home.manage
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Checkbox
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.BasePreview
+import ca.bc.gov.bchealth.compose.MyHealthTheme
+import ca.bc.gov.bchealth.compose.MyHealthTypography
+import ca.bc.gov.bchealth.compose.bold
+import ca.bc.gov.bchealth.compose.theme.primaryBlue
+import ca.bc.gov.bchealth.compose.theme.statusBlue
+import ca.bc.gov.bchealth.compose.theme.white
+import ca.bc.gov.bchealth.ui.home.QuickAccessTileItem
+
+@Composable
+fun QuickAccessManagementScreen(
+    viewModel: QuickAccessManagementViewModel,
+    onClickItem: (QuickAccessTileItem) -> Unit,
+    onUpdateCompleted: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val uiState = viewModel.uiState.collectAsState().value
+
+    LaunchedEffect(Unit) {
+        viewModel.loadTilesUi()
+    }
+
+    QuickAccessManagementContent(uiState, onClickItem, modifier)
+
+    if (uiState.isUpdateCompleted) {
+        onUpdateCompleted.invoke()
+    }
+}
+
+@Composable
+private fun QuickAccessManagementContent(
+    uiState: QuickAccessManagementViewModel.QuickAccessManagementUiState,
```

Comment:
This will trigger recomposition every time, ui state changes.
I would suggest pass only param require for the content screen.

Reply:
done